### PR TITLE
Extract messages

### PR DESCRIPTION
**Pull request by turkeylurkey**

### CODE REVIEW

File: jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/servlet/HttpServletQuickFix.java [hunk 1]
```diff
@@ -51,14 +52,14 @@ public List<? extends CodeAction> getCodeActions(JavaCodeActionContext context,
...
-            final String TITLE_MESSAGE = "Let ''{0}'' extend ''{1}''";
-            String args[] = { BasicElementLabels.getJavaElementName(parentType.getName()),
-                    BasicElementLabels.getJavaElementName(ServletConstants.HTTP_SERVLET) };
-            ChangeCorrectionProposal proposal = new ExtendClassProposal(MessageFormat.format(TITLE_MESSAGE, args),
-                    context.getCompilationUnit(), parentType, context.getASTRoot(), "jakarta.servlet.http.HttpServlet",
-                    0);
+            String title = Messages.getMessage("LetClassExtend", 
+                    BasicElementLabels.getJavaElementName(parentType.getName()), 
+                    BasicElementLabels.getJavaElementName(ServletConstants.HTTP_SERVLET));
+            ChangeCorrectionProposal proposal = new ExtendClassProposal(title,
```
Comment (mezarin) [inline jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/servlet/HttpServletQuickFix.java:58]: Remove: `import java.text.MessageFormat;`.
No longer used in this file.

**Review by mezarin — COMMENTED** [review]
Hi Paul, the changes look good. I made a couple of comments.

File: jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/cdi/ManagedBeanDiagnosticsCollector.java [hunk 2]
```diff
@@ -107,8 +108,7 @@ public void collectDiagnostics(ICompilationUnit unit, List<Diagnostic> diagnosti
                     if (isManagedBean && Flags.isPublic(fieldFlags) && !Flags.isStatic(fieldFlags)
                             && (fieldScopes.size() != 1 || !fieldScopes.get(0).equals(DEPENDENT_FQ_NAME))) {
                         diagnostics.add(createDiagnostic(field, unit,
-                                createAnnotationDiagnostic(DEPENDENT,
-                                        "be the only scope defined by a managed bean with a non-static public field."),
+                                Messages.getMessage("ManagedBeanWithNonStaticPublicField"),
```
[the remark this answers is not in the document]
Comment (turkeylurkey) [reply jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/cdi/ManagedBeanDiagnosticsCollector.java:111]: I agree. I noticed this too but it was too much to process at once. I checked a style guide that suggested periods on messages and not on titles. I'll make another pass now.

File: jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/codeAction/proposal/ExtendClassProposal.java [hunk 3]
```diff
@@ -37,8 +37,6 @@
 
 public class ExtendClassProposal extends ChangeCorrectionProposal {
```
[the remark this answers is not in the document]
Comment (turkeylurkey) [reply jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/codeAction/proposal/ExtendClassProposal.java:37]: Ok.

File: jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/cdi/ManagedBeanDiagnosticsCollector.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (turkeylurkey) [reply jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/cdi/ManagedBeanDiagnosticsCollector.java:111]: Updated diagnostic messages, more research required on quick fix labels.

File: jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/servlet/HttpServletQuickFix.java (hunk 1 above)
Comment (turkeylurkey) [reply jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/servlet/HttpServletQuickFix.java:57]: Ok.

File: jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/cdi/ManagedBeanDiagnosticsCollector.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (turkeylurkey) [reply jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/cdi/ManagedBeanDiagnosticsCollector.java:111]: Other quick fix labels (MicroProfile and Java) do not use sentence format and do not end with a period.

**Pull request merged**